x/orderbook/bucket_test.go

```diff
@@ -1,138 +1,289 @@
 package orderbook
 
 import (
-	"time"
+	"encoding/binary"
 	"testing"
+	"time"
 
 	"github.com/iov-one/weave"
 	"github.com/iov-one/weave/coin"
 	"github.com/iov-one/weave/errors"
 	"github.com/iov-one/weave/orm"
 	"github.com/iov-one/weave/weavetest"
+	"github.com/iov-one/weave/weavetest/assert"
 )
 
 func TestMarketIDindexer(t *testing.T) {
+	marketID := weavetest.SequenceID(5)
```

Comment:
Yes, nice improvement this one

Reply:
I was having a silly error totally caused by me. Left this change by mistake :smiley: